client/blocks/hello-vote-notice/index.jsx

```diff
@@ -70,6 +72,10 @@ export default connect(
 				isSiteSection( state ) &&
 				'settings' !== getSectionName( state ) &&
 				canCurrentUser( state, selectedSiteId, 'manage_options' ) &&
+				(
+					! userStartDate ||
+					moment( userStartDate ).isBefore( moment().subtract( 1, 'weeks' ) )
```

Comment:
You may actually be able to get moment the `connect` selector [`mapStateToProps` second argument, `ownProps`](https://github.com/reactjs/react-redux/blob/master/docs/api.md#connectmapstatetoprops-mapdispatchtoprops-mergeprops-options), since it's passed by [`localize`](https://github.com/Automattic/i18n-calypso#localize). Dunno if there's really any benefit in doing so, though.